models/balancer/arbitrum/balancer_v2_arbitrum_protocol_fee.sql

```diff
@@ -0,0 +1,23 @@
+{% set blockchain = 'arbitrum' %}
+
+{{
+    config(
+        schema='balancer_v2_' + blockchain,
+        alias = 'protocol_fee', 
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['day', 'pool_id', 'token_address'],
+        post_hook='{{ expose_spells(\'["arbitrum"]\',
```

Comment:
can we remove chain-specific spell post hooks? i imagine you plan to use the cross-chain one on the app, that'll help keep the data explorer clean
(again, will only comment here, but applies to all blockchain versions)